### PR TITLE
Id updates

### PR DESCRIPTION
**Pull request by steven1046**

### CODE REVIEW

File: json-guides/microprofile-config.json [hunk 1]
```diff
@@ -11,18 +11,18 @@
...
             "description": [
-              "Explore how to inject configuration data into a microservice without repackaging the application each time the underlying runtime environment changes.  <a href='https://microprofile.io/' target='_blank'>MicroProfile</a> makes building configurable microservices easy with its <a href='https://github.com/eclipse/microprofile-config' target='_blank'>MicroProfile Config</a> feature. Application configuration properties from multiple sources are combined into a single set of configuration properties and accessed from a single API by your application.<br>",
+              "Explore how to inject configuration data into a microservice without repackaging the application each time the underlying runtime environment changes.  <a href='https://microprofile.io/' target='_blank'>MicroProfile</a> makes building configurable microservices easy with its <a href='https://github.com/eclipse/microprofile-config' target='_blank'>MicroProfile Config</a> feature. Your application combines  application configuration properties from multiple sources are combined into a single set and accesses the properties from a single API by your application.<br>",
```
Comment (erica-banda-03) [inline json-guides/microprofile-config.json]: I think you have some extra words in this sentence "Your application combines application..."
Shouldn't it be, "Your application combines application configuration properties from multiple sources into a single set and accesses the properties from a single API."

File: json-guides/microprofile-config.json [hunk 2]
```diff
@@ -253,15 +253,15 @@
...
+                "In Open Liberty, you can use <code>server.env</code> files at the installation and server levels to specify environment variables. The installation level file is located at <code>${wlp.install.dir}/etc/server.env</code>, and the server level is located at <code>${server.config.dir}/server.env</code>. If both files exist, the contents of the two files are merged. If the same environment variable is in both files, the value in the server level file takes precedence over the value in the installation level file. Use the <code>${server.config.dir}/server.env</code> file to configure the <code>port</code> environment variable.",
```
Comment (erica-banda-03) [inline json-guides/microprofile-config.json]: Should we add "file" after "server level" in this sentence?

File: json-guides/microprofile-config.json [hunk 3]
```diff
@@ -92,12 +92,12 @@
             "name": "ConfigureViaInject",
             "title": "Injecting configuration through @ConfigProperty annotation",
             "description": [
-                "After you modify your <code>server.xml</code> file to <a href='#enabling-microprofile-config'>include the mpConfig feature</a>, a single configuration property value can be injected directly into your code using the <code>@Inject</code> and the <code>@ConfigProperty</code> annotations. The injected value is static and therefore will not change after the application starts.",
+                "After you modify your <code>server.xml</code> file to <a href='#enabling-microprofile-config'>include the mpConfig feature</a>, inject a single configuration property value can be injected directly into your code with the <code>@Inject</code> and the <code>@ConfigProperty</code> annotations. The injected value is static and does not change after the application starts.",
```
[the remark this answers is not in the document]
Comment (steven1046) [reply json-guides/microprofile-config.json]: done

File: json-guides/microprofile-config.json (hunk 1 above)
Comment (steven1046) [reply json-guides/microprofile-config.json]: done

**Pull request merged**